### PR TITLE
Removes MSay

### PR DESCRIPTION
**Pull request by Zonespace27**
# About the pull request
Removes msay, gives all staff permission to use asay

image

# Explain why it's good for the game
Overall, having both msay + asay feels just plain redundant
There's very few situations where you need to obscure something from all mods that can't be done just the same via discord

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image

</details>

# Changelog
:cl:
admin: Removed msay. All staff now have access to asay
/:cl:

### CODE REVIEW

**Review by realforest2001 — APPROVED** [review]
Approved from host side of things.

**Comment by BroseidonMOO** [comment]
Please reverse this, and remove asay instead of msay. The vast majority of Staff are used to using msay, not asay.

**Comment by Zonespace27** [comment]
> Please reverse this, and remove asay instead of msay. The vast majority of Staff are used to using msay, not asay.

330a9cd10d2b08e7389a4c2563668c4135241df4

**Review by harryob — APPROVED** [review]

**Review by Mister-moon1 — COMMENTED** [review]
could the colour of asay be changed to #74471b like the old msay, the colour is softer and easier on the eyes compared to the harsh purple as is right now

File: tgui/packages/tgui-say/styles/colors.scss [hunk 1]
```diff
@@ -14,7 +14,6 @@ $comms: #b4b4b4;
 $me: #5975da;
 $ooc: #1c52f5;
 $looc: #e362b4;
-$msay: #74471b;
 $mentor: #b5850d;
 $asay: #9611d4;
```
Comment (Mister-moon1) [inline tgui/packages/tgui-say/styles/colors.scss:18]: colour in question to be changed to #74471b

**Pull request merged**